Source/Common/Config.cpp

```diff
@@ -168,6 +168,7 @@ MainLoader::MainLoader(FEXCore::Config::LayerType Type, const char* ConfigFile)
   , Config {ConfigFile} {}
 
 void MainLoader::Load() {
+  LogMan::Msg::DFmt("[MainLoader] Loading config file: {}", Config.c_str());
```

Comment:
This message goes nowhere at application launch since the LogManager handlers aren't setup at the start so its outputFD ends up being -1 so the writes do EBADFD. Same as the apploader message below.

Reply:
I am slightly confused about this. I am probably missing something but I can see these messages being output to server. There are no errors. In which case is the LogManager not set up when we load the configs?

Reply:
Interesting, shouldn't do. The default logmanager handler is setup by time this loads but it defaults to SilentLog (through uninitialized bool but eh) and default output file of -1. stracing showed these trying to write to -1 locally

Installer getting installed:
https://github.com/FEX-Emu/FEX/blob/main/Source/Tools/FEXLoader/FEXLoader.cpp#L275
LogManager getting switched over to Server significantly later:
https://github.com/FEX-Emu/FEX/blob/main/Source/Tools/FEXLoader/FEXLoader.cpp#L346

Reply:
OK, maybe we are doing things differently... I start a FEXServer with `-f -p`. Then I start FEX and get in the FEXServer window:
```
[DEBUG] [MainLoader] Loading config file: /FEX/FEXDeb-d8c2fcb77/share/fex-emu/Config.json
[DEBUG] [MainLoader] Loading config file: /home/steamvr/.fex-emu/Config.json
[DEBUG] [FEXServer] New connection accepted
[INFO][436755.473149275][242833.242833] [242833][steam-launch-wrapper] Sleeping for 5 seconds
[INFO][436760.623612865][242833.242833] IR-pre 0x7f9c2e3de0:
```

which... is quite useful tbh.

Reply:
We would need to change how logging occurs if we want to block debug message on release builds, since a potential for 3 messages in the server log when pressure-vessel and proton spin up hundreds of processes isn't viable.

Reply:
OK - lets close this for now then.